lxd/network/network_utils_test.go

```diff
@@ -86,3 +88,52 @@ func Example_parseIPRange() {
 	// Start: fd22:c952:653e:ffff::1, End: fd22:c952:653e:ffff::ffff
 	// Start: ::aaaa:ffff:ffff:ffff:1, End: ::aaaa:ffff:ffff:ffff:ffff
 }
+
+func Example_IPRangesOverlap() {
```

Comment:
Apparently Go doesn't let you have a capital letter after the `Example_` prefix.

`Example_ipRangesOverlap` should be fine.

Reply:
I think it maybe that the underscore is the issue, E.g.https://github.com/lxc/lxd/blob/master/lxd/refcount/refcount_test.go